cap/modules/schemas/models.py

```diff
@@ -194,6 +199,27 @@ def validate_name(self, key, name):
             )
         return name
 
+    @validates('config')
+    def validate_config(self, key, value):
+        """Validate the config."""
+        if value:
+            config_data = None
+            with open('cap/modules/schemas/config.json') as config_json:
```

Comment:
maybe we should move the definition of the validator outside this method, maybe in the initialization of the schema module - shouldn't run every time